tests/bin.js

```diff
@@ -8,45 +8,47 @@ const { version } = require('../package.json')
 
 const { FIXTURES_DIR } = require('./helpers/main')
 
-const BINARY_PATH = `${__dirname}/../src/bin.js`
+const BINARY_PATH = join(__dirname, '..', 'dist', 'bin.js')
+
+const exec = (args, options) => execa('node', [BINARY_PATH, ...args], options)
```

Comment:
We need the file to be executable. I don't think npm automatically sets execution permissions on files specified in `bin`, so presumably this works right now because we set those permissions on the file ourselves. If the file is going to be created as part of the build step, how do we handle that? 🤔

Reply:
Hmm, I'd have assumed that specifying it under package.json's `bin` field is enough, and NPM will automatically make it executable on install. I'll cut a prerelease of this branch to verify that.

Reply:
```bash
❯ npm i --save-global @netlify/zip-it-and-ship-it@4.23.3-setup-typescript-1  
...
❯ zip-it-and-ship-it --version
4.23.3-setup-typescript-1
```

Works on my machine 😄